api/internal/git/repospec_test.go

```diff
@@ -213,14 +232,14 @@ func TestNewRepoSpecFromUrl_Smoke(t *testing.T) {
 			},
 		},
 		{
-			name:      "non-github_scp",
+			name:      "non-github_scp with path delimiter",
 			input:     "git@bitbucket.org:company/project.git//path?ref=branch",
 			cloneSpec: "git@bitbucket.org:company/project.git",
 			absPath:   notCloned.Join("path"),
 			repoSpec: RepoSpec{
 				Host:         "git@bitbucket.org:",
 				RepoPath:     "company/project",
-				KustRootPath: "/path",
+				KustRootPath: "path",
```

Comment:
I don't think it matters whether we end up with a leading slash here, but previously we were inconsistent. Github URLs (which generally are best tested/supported) did not have the prefix previously, so I took them all in that direction.